### PR TITLE
Added a basic lorem ipsum generator command

### LINKED ISSUE #23: Lorem ipsum generator
Pretty self-explanatory, but this is basically what I had in mind:
https://github.com/raarellano/alfred-lorem-ipsum-workflow

### PR DESCRIPTION
**Pull request by DaveWM**
I've had a go at creating a basic lorem ipsum generator command. It's pretty barebones, but the idea is that you enter something like `lipsum 5 sentences` and it copies 5 sentences to the clipboard. It uses a single lorem ipsum paragraph to generate the output text, and will loop round if necessary (e.g. if you do `lipsum 2 paragraphs` it will repeat the text twice).

Resolves #23

### CODE REVIEW

**Comment by DaveWM** [comment]
Thanks @reibitto :). I've just pushed another commit allowing the "word/sentence/paragraph" argument to be singular. 

I had a go at allowing a command like `lorem sentence`, but couldn't get it working (I'll add another comment with the code I tried). A help command would be nice, but it seems like that should be handled in a generic way (in `Command` maybe?) rather than per-command. Once I'm a bit more familiar with the codebase, I can have a look into how we could do that :)

File: core/src/main/scala/commandcenter/command/LoremIpsumCommand.scala [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+  val numOpt = Opts.argument[Int]("number").withDefault(1)
+  val typeOpt = Opts.argument[String]("words, sentences, or paragraphs").mapValidated {
+    case arg if arg.matches("words?") => Validated.valid(Word)
+    case arg if arg.matches("sentences?") => Validated.valid(Sentence)
+    case arg if arg.matches("paragraphs?") => Validated.valid(Paragraph)
+    case s => Validated.invalidNel(s"${s} is not valid: should be 'words', 'sentences', or 'paragraphs'.")
+  }.withDefault(Paragraph)
+
+  val lipsumCommand = decline.Command(title, title)((numOpt, typeOpt).tupled)
```
Comment (DaveWM) [inline core/src/main/scala/commandcenter/command/LoremIpsumCommand.scala:39]: I tried to update this to allow commands like `lorem sentence` by changing this line to:
```scala
val lipsumCommand = decline.Command(title, title)((numOpt, typeOpt).tupled.orElse(typeOpt.map((1, _))))
```
That doesn't work unfortunately - typing in `lorem sentence` gives you an error message of "Invalid integer: sentence".

**Comment by reibitto** [comment]
Thanks. Looks good to merge.

I think it's fine to not worry about input like `lorem sentence`. The current usage looks good to me.

The other idea I was contemplating was the multiple results thing. It should be possible now that this bug was fixed a little while ago:
https://github.com/reibitto/command-center/issues/35

But that can always be added later. I think I'm going to concentrate on #39 to and then add tests before trying to do anything too complicated with individual commands.

File: core/src/main/scala/commandcenter/command/LoremIpsumCommand.scala (hunk 1 above)
Comment (reibitto) [reply core/src/main/scala/commandcenter/command/LoremIpsumCommand.scala:39]: Right, those are positional arguments so an Int argument alone wouldn't be able to cover both cases. You would either need to change the argument to a String and validate from there. Or possibly try making it an Either. But I think that might overcomplicate things.

**Pull request merged**